prow/plugins/require-matching-label/require-matching-label.go

```diff
@@ -95,16 +100,49 @@ func helpProvider(config *plugins.Configuration, _ []config.OrgRepo) (*pluginhel
 	if err != nil {
 		logrus.WithError(err).Warnf("cannot generate comments for %s plugin", pluginName)
 	}
-	return &pluginhelp.PluginHelp{
+	pluginHelp := &pluginhelp.PluginHelp{
 			Description: `The require-matching-label plugin is a configurable plugin that applies a label to issues and/or PRs that do not have any labels matching a regular expression. An example of this is applying a 'needs-sig' label to all issues that do not have a 'sig/*' label. This plugin can have multiple configurations to provide this kind of behavior for multiple different label sets. The configuration allows issue type, PR branch, and an optional explanation comment to be specified.`,
 			Config: map[string]string{
 				"": fmt.Sprintf("The plugin has the following configurations:\n<ul><li>%s</li></ul>", strings.Join(descs, "</li><li>")),
 			},
 			Snippet: yamlSnippet,
-		},
-		nil
+		}
+	pluginHelp.AddCommand(pluginhelp.Command{
+		Usage:       "/check-require-labels",
+		Description: "Forces the 'require-matching-label' plugin to reprocess the pull request.",
+		Featured:    true,
+		WhoCanUse:   "Anyone",
+		Examples:    []string{"/check-require-labels"},
+	})
+	return pluginHelp, nil
+
 }
 
+func handleCommentEvent(pc plugins.Agent, ce github.GenericCommentEvent, pre github.PullRequestEvent) error {
```

Comment:
This is the wrong method signature for a GenericCommentEvent handler, there should not be a PullRequestEvent parameter.